pilot/pkg/networking/util/util.go

```diff
@@ -264,6 +264,9 @@ func ResolveHostsInNetworksConfig(config *meshconfig.MeshNetworks) {
 			gwIP := net.ParseIP(gw.GetAddress())
 			if gwIP == nil {
 				addrs, err := net.LookupHost(gw.GetAddress())
+				if err != nil {
+					log.Warnf("error resolving host %#v: %v", gw.GetAddress(), err)
+				}
```

Comment:
Is this called only from bootstrap? Else it will flood the logs.

Reply:
Also called when filewatcher detects file change

Reply:
I think those two cases are fine - I wouldn't expect the meshNetworks config file to be changing often. And when it does - I'd like to see if host resolution fails then too